src/Extension/FluentExtension.php

```diff
@@ -997,4 +997,17 @@ protected function detectLocalisedTableField($tables, $sql)
 
         return [null, null, false];
     }
+
+    /**
+     * Returns the native language name for the selected locale/language
+     *
+     * @return string
+     */
+    public function getSelectedLanguage()
+    {
+        /** @var ArrayData $information */
+        $information = $this->LocaleInformation(FluentState::singleton()->getLocale());
+
+        return $information->LanguageNative;
```

Comment:
Instead of adding a method that only accesses this single field, why don't we return `$information` and let you pick what field you use in the template?

Suggest to name this method `SelectedLocaleInformation` (same as `LocaleInformation` but in selected locale).